docs/dev/python/index.md

```diff
@@ -2,7 +2,7 @@
 title: "Python API"
 nav-id: python
 nav-parent_id: dev
-nav-pos: 50
```

Comment:
Why change this? This changes moves `Python API` page between `DataStream API` page and `DataSet API` page. Is this intended?